src/cartservice/.dockerignore

```diff
@@ -0,0 +1,8 @@
+**/*.sh
+**/*.bat
```

Comment:
why these files are ignored?

Reply:
Those type of files should not go the container itself (at least in the context of the `cartservice`), so to avoid any `COPY` command in the `Dockerfile` bringing them, we could add their exclusion in the `.dockerignore` file as a best practice.

Reply:
Agree on the bat files for the container but not necessarily for sh files. Let's not ignore *.sh - in case we might want to wrap the microservice executable in a script

Reply:
Done.